doc/source/development/extending.rst

```diff
@@ -208,6 +208,194 @@ will
 2. call ``result = op(values, ExtensionArray)``
 3. re-box the result in a ``Series``
 
+:class:`~pandas.api.extensions.ExtensionArray` Series Operations Support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 0.25.0
+
+In addition to operators like `__mul__` and `__add__`, the pandas Series
+namespace provides a long list of useful operations such as :meth:`Series.round`,
+:meth:`Series.sum`, :meth:`Series.abs`, etc'. Some of these are handled by
+pandas own algorithm implementations (via a dispatch function), while others
+simply call an equivalent numpy function with data from the underlying array.
+In order to support this operations in a new ExtensionArray, you must provide
+an implementation for them.
+
+.. note::
+
+    There is a third category of operation which live on the `pandas`
+    namespace, for example `:meth:pd.concat`. There is an equivalent
+    numpy function `:meth:np.concatenate`, but this is not used. n
+    general, these function should just work with your EA, you do not
+    need to impelment more than the general EA interface.
+
+As of 0.25.0, the list of series operations which pandas' provides its own
+implementations are: :meth:`Series.any`, :meth:`Series.all`,
+:meth:`Series.min`, :meth:`Series.max`, :meth:`Series.sum`,
+:meth:`Series.mean`, :meth:`Series.median`, :meth:`Series.prod`
+(and its alias :meth:`Series.product`), :meth:`Series.std`,
+:meth:`Series.var`, :meth:`Series.sem`,
+:meth:`Series.kurt`, and :meth:`Series.skew`.
+
+In order to implement any of this functions, your ExtensionArray must include
+an Implementation of :meth:`ExtensionArray._reduce`. Once you provide an
+implementation of :meth:`ExtensionArray._reduce` which handles a particular
+method, calling the method on the Series will invoke the implementation
+on your ExtensionArray. All these methods are reduction functions, and
+so are expected to return a scalr value of some type. However it is perfectly
+acceptable to return some instance of an :class:`pandas.api.extensions.ExtensionDtype`.
```

Comment:
I don't think we want to recommend that though?

Reply:
should be ExtensionArray.

Reply:
fixed.

Reply:
Can you give an example where a reduction can return an ExtensionArray?

Reply:
of course not. oh, I see what you mean now. That's the wrong place for that sentence. Moved it into a note,
to state explicitly that overrides of numpy functions are not required to return numpy arrays, but are allowed
and usually will reutrn EA instances.